src/room/GroupCallView.tsx

```diff
@@ -279,6 +279,7 @@ export function GroupCallView({
           setUserChoices(choices);
           enter();
         }}
+        muteAudio={participants.size > 8}
```

Comment:
It looks like each time the participant count changes (and the participant count is more than 8), the user will get muted, is it an expected behavior? - I.e. if you have 7 participants in a room, and you're in a lobby, unmuted, then suddenly the UI gets muted (because the 8th participant joined the call), at that point you may unmute yourself and then get muted again (9th participant joined).

I believe the check for participant size is something that we probably want to do only once upon joining.

Reply:
Yeah, I see what you mean though I wonder what exactly is the desired behaviour here... R previously said:

> I think the desired behavior that we discussed was to reflect this automatic mute state in the lobby, so it's more apparent to the user what their mute state in the call will be. Is that easily done?

Reply:
Yeah, it makes sense, but what I meant to say is that we probably don't want to change the mute state each time when the amount of participants changes as it means that if you unmute the video, it will get muted automatically and each time you unmute it, it will get muted again once someone in the room joins (which won't be obvious for the lobby user).

I remember that some other solution does it like this: when you join the room, your devices are muted (and there is a tooltip telling you that), but if you unmute yourself, you don't get muted once someone new joins. I.e. I think the amount of participants and setting a mute state should probably be done once, not each time someone enters/leaves a call. Does it make any sense? :)

Reply:
Yeah, I think so. Does [this](https://github.com/vector-im/element-call/pull/1306) make sense?